bin/check-topics-name.rb

```diff
@@ -0,0 +1,61 @@
+#! /usr/bin/env ruby
+#
+# check-topics-name
+#
+# DESCRIPTION:
+#   This plugin checks topic's name.
+#
+# OUTPUT:
+#   plain-text
+#
+# PLATFORMS:
+#   Linux
+#
+# DEPENDENCIES:
+#   gem: sensu-plugin
+#   gem: zookeeper
+#
+# USAGE:
+#   ./check-topics-name
+#
+# NOTES:
+#
+# LICENSE:
+#   Olivier Bazoud
+#   Released under the same terms as Sensu (the MIT license); see LICENSE
+#   for details.
+#
+
+require 'sensu-plugin/check/cli'
+require 'json'
+require 'zookeeper'
+
+class TopicsCheck < Sensu::Plugin::Check::CLI
+  option :zookeeper,
```

Comment:
same question about if this is the right place for this check or if it should live in the zookeeper plugin.